cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -430,10 +431,57 @@ func (scaleSet *ScaleSet) buildNodeFromTemplate(template compute.VirtualMachineS
 
 	// GenericLabels
 	node.Labels = cloudprovider.JoinStringMaps(node.Labels, buildGenericLabels(template, nodeName))
+	// Labels from the Scale Set's Tags
```

Comment:
For these values, do we want to enable the users to use `/` in the label or taint keys?